### ISSUE TITLE
New docker tags for springBoot features

### ISSUE THREAD

**Issue by tjwatson** [WASdev/ci.docker#166]
Once the Spring Boot features are included in an Websphere Liberty release (https://github.com/OpenLiberty/open-liberty/issues/3169) we need to create new tags in docker that include the new features for Spring Boot.

**Comment by mrglavas** [comment]
@dazavala @tjwatson: I've merged a PR: https://github.com/WASdev/ci.docker/pull/177 on to master. I got a good local build and the Travis build was green. You can use this for further testing. Can spin a new PR if there are any updates needed.

**Comment by arthurdm** [comment]
thanks guys.     Please let me know once this is ready to be published and I can make the PR to Docker Hub.

**Comment by dazavala** [comment]
Mike, the two springBoot-x images work as expected.

**Comment by mrglavas** [comment]
@dazavala Thanks.
@arthurdm I believe we're ready to publish to Docker Hub now.

**Comment by arthurdm** [comment]
Cool - PR created: https://github.com/docker-library/official-images/pull/4609    We'll need to watch that PR for any questions the Docker Hub folks have.  

Great work guys!

**Comment by dazavala** [comment]
Hello, @mrglavas.  The official websphere-liberty springBoot tags and readme additions are published at docker hub.  Let's close this issue.

**Comment by mrglavas** [comment]
@dazavala Thanks. Closing this one.
